### PR TITLE
update wallet docs for Moonbeam

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR includes changes to the wallet section for connecting to Moonbeam. ~I added in some directions to add Moonbeam as a custom network for MathWallet as I figured that would be better than nothing, but I'm not sure if others would agree. So I figured I would let this PR go through an initial review before creating and editing the screenshots I would need for that section. So that's why there are some image placeholders there right now.~

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

n/a

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove & reorder images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

I added a few images to the `mathwallet.md` and `metamask.md` pages. Those images are postfixed with `-new`. After translations are done, a PR will need to be opened in the `moonbeam-docs` repo to:
 - rearrange the `mathwallet.md` images to be in numeric order as you go down the page
   -  remove the `-new` postfix from images 2 & 3
- for `metamask.md` images the original image 6 & 7 will need to be deleted
  - remove the `-new` postfix from image 6
  - reorder all images after 6

If you have any questions about the required changes, let me know! 🙂

### CODE REVIEW

File: tokens/connect/mathwallet.md [hunk 1]
```diff
@@ -9,42 +9,67 @@ description: This guide walks you through how to connect Mathwallet, a browser-b
...
+
+![Connect to Moonbase Alpha](/images/tokens/connect/mathwallet/mathwallet-3.png)
+
+And that is it, you now have MathWallet connected to the Moonbase Alpha TestNet! Your wallet should look like this:
+
+![Wallet Connected to Moonbase Alpha](/images/tokens/connect/mathwallet/mathwallet-4.png)
+
 ## Adding a Wallet {: #adding-a-wallet } 
 
 The following steps will show you how to interact with the Moonbase Alpha TestNet, but can also be used to interact with Moonriver.
```
Comment (albertov19) [inline tokens/connect/mathwallet.md:75]: And Moonbeam as well :)

File: tokens/connect/mathwallet.md [hunk 2]
```diff
@@ -9,42 +9,67 @@ description: This guide walks you through how to connect Mathwallet, a browser-b
...
+![Wallet Connected to Moonbeam]()
+
+Eventually this process will be improved upon, and it will be similar to connecting to Moonriver.
 
 ## Connect to Moonriver {: #connect-to-moonriver } 
 
-Getting started with Moonriver is a straightforward process. All you have to do is click Switch Network and select Moonriver
+Getting started with Moonriver is a straightforward process. All you have to do is click **Switch Network** and select **Moonriver**
 
 ![Connect to Moonriver](/images/tokens/connect/mathwallet/mathwallet-5.png)
```
Comment (albertov19) [inline tokens/connect/mathwallet.md:53]: This image and number 6 need to be updated with the Moonriver Logo

File: tokens/connect/mathwallet.md [hunk 3]
```diff
@@ -9,42 +9,67 @@ description: This guide walks you through how to connect Mathwallet, a browser-b
...
 
 And that is it, you now have MathWallet connected to Moonriver! Your wallet should look like this:
 
 ![Wallet Connected to Moonriver](/images/tokens/connect/mathwallet/mathwallet-6.png)
 
+## Connect to Moonbase Alpha {: #connect-to-moonbase-alpha } 
+
+In this part, we'll go through the process of connecting MathWallet to Moonbase Alpha. Enable Moonbase Alpha under Settings (top right gear icon) -> Networks -> Ethereum.
```
Comment (albertov19) [inline tokens/connect/mathwallet.md:61]: We never do the `-> ->` thing I think. Not sure what you want to do here

File: tokens/connect/metamask.md [hunk 4]
```diff
@@ -38,15 +38,13 @@ Learn [how to integrate a Connect MetaMask button](/builders/interact/metamask-d
 <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/ywpc1UwpIyg' frameborder='0' allowfullscreen></iframe></div>
 <style>.caption { font-family: Open Sans, sans-serif; font-size: 0.9em; color: rgba(170, 170, 170, 1); font-style: italic; letter-spacing: 0px; position: relative;}</style><div class='caption'>In this video, we'll show you how to connect your MetaMask wallet to the Moonriver network</a></div>
 
-
```
Comment (albertov19) [inline tokens/connect/metamask.md:41]: I'm guessing the Connect to Moonbeam button is added in the other PR, just worth mentioning to make sure :)

File: tokens/connect/metamask.md [hunk 5]
```diff
@@ -38,15 +38,13 @@ Learn [how to integrate a Connect MetaMask button](/builders/interact/metamask-d
 <style>.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }</style><div class='embed-container'><iframe src='https://www.youtube.com/embed/ywpc1UwpIyg' frameborder='0' allowfullscreen></iframe></div>
```
Comment (albertov19) [inline tokens/connect/metamask.md:38]: It seems that one of the videos is not available anymore

**Review by albertov19 — CHANGES_REQUESTED** [review]

File: tokens/connect/metamask.md (hunk 4 above)
Comment (eshaben) [reply tokens/connect/metamask.md:41]: yes it is!

File: tokens/connect/metamask.md (hunk 5 above)
Comment (eshaben) [reply tokens/connect/metamask.md:38]: this was one of the fixes included in the PR I merged last night 🙂 

I'll merge master into this PR!

**Comment by eshaben** [comment]
this is ready for review again

**Review by albertov19 — APPROVED** [review]

**Comment by albertov19** [comment]
LGTM 🔥

**Pull request merged**